src/main/java/core/basesyntax/strategy/impl/PurchaseOperationImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class PurchaseOperationImpl implements OperationHandler {
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() <= MIN_QUANTITY) {
```

Comment:
The constant MIN_QUANTITY is not defined in this class. You should define it or make sure it is properly imported if it's declared elsewhere.